<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopWindow</class>
 <widget class="QMainWindow" name="TopWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0227273 rgba(255, 255, 255, 255), stop:1 rgba(59, 59, 59, 162));</string>
      </property>
      <widget class="QPushButton" name="SerialpushButton">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>130</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;华文中宋&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>串口</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="TCPpushButton">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>230</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;华文中宋&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>TCP</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="USBpushButton">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>330</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>华文中宋</family>
         <pointsize>22</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;华文中宋&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>USB</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <zorder>TCPpushButton</zorder>
      <zorder>SerialpushButton</zorder>
      <zorder>USBpushButton</zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCOM">
   <property name="text">
    <string>COM</string>
   </property>
  </action>
  <action name="actionTCP_UDP">
   <property name="text">
    <string>TCP&amp;UDP</string>
   </property>
  </action>
  <action name="actionUSB">
   <property name="text">
    <string>USB</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
